### PR TITLE
feat: add support for Django 3.2

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Adds support for Django 3.2
- Remove Travis runners
- Run tests on Github Actions
- Fix Django 3.2 failures
- Fix quality failures

**JIRA:** https://openedx.atlassian.net/browse/BOM-2646 

**Steps Followed:** https://openedx.atlassian.net/wiki/spaces/AC/pages/1704591765/Django+3.2+Upgrade+Steps+to+Upgrade+a+Repository

### CODE REVIEW

**Review by awais786 — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+    - name: Install requirements
+      run: pip install -r requirements/test.txt
+    - name: Upgrade packages
+      run: pip install -U pip wheel codecov
```
Comment (mraarif) [inline .github/workflows/ci.yml:26]: we should instead have a `pip.txt` that contains updated version of pip, wheel and setuptools and codecov should be part of test dependencies

**Review by mraarif — APPROVED** [review]

**Review by Zacharis278 — APPROVED** [review]
lgtm

**Review by schenedx — APPROVED** [review]
The change is fine with me.
Just want to clarify the goal of this PR. Add support to Django 3.2 means you have added testing and tox execution for django 3.2. 
That does not mean you are upgrading into django 3.2 in this PR, right?
If so, the PR looks good.

**Comment by iamsobanjaved** [comment]
Yes, added tests for Django 3.2, will upgrade to Django 3.2 later

**Pull request merged**